tests/test_docs.py

```diff
@@ -0,0 +1,31 @@
+import doctest
```

Comment:
If we use the `--doctest-glob` options as in the github workflows and also use `--doctest-modules`, I don't think we need this explicit file to collect doctests. By including it, we might actually run duplicates, though maybe pytest ignores those?